blocks/init/src/Blocks/components/button/button.php

```diff
@@ -49,46 +58,34 @@
 	Components::selector($buttonIsAnchor, 'js-scroll-to-anchor'),
 ]);
 
+$buttonTag = $buttonUrl ? 'a' : 'button';
 ?>
 
-<?php if (! $buttonUrl) { ?>
-	<button
-		class="<?php echo esc_attr($buttonClass); ?>"
-		<?php if (!empty($buttonId)) { ?>
-			id="<?php echo esc_attr($buttonId); ?>"
-		<?php } ?>
-		title="<?php echo esc_attr($buttonContent); ?>"
-		<?php if (!empty($buttonAriaLabel)) { ?>
-			aria-label="<?php echo esc_attr($buttonAriaLabel); ?>"
-		<?php } ?>
-		data-id="<?php echo esc_attr($unique); ?>"
-		<?php
-		foreach ($buttonAttrs as $key => $value) {
-			echo wp_kses_post("{$key}=" . $value . " ");
-		}
-		?>
-	>
-		<?php echo esc_html($buttonContent); ?>
-	</button>
-
-<?php } else { ?>
-	<a
-		href="<?php echo esc_url($buttonUrl); ?>"
-		class="<?php echo esc_attr($buttonClass); ?>"
-		<?php if (!empty($buttonId)) { ?>
-			id="<?php echo esc_attr($buttonId); ?>"
-		<?php } ?>
-		title="<?php echo esc_attr($buttonContent); ?>"
-		<?php if (!empty($buttonAriaLabel)) { ?>
-			aria-label="<?php echo esc_attr($buttonAriaLabel); ?>"
-		<?php } ?>
-		data-id="<?php echo esc_attr($unique); ?>"
-		<?php
-		foreach ($buttonAttrs as $key => $value) {
+<?php
+	// phpcs:ignore Eightshift.Security.ComponentsEscape.OutputNotEscaped
+	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest);
```

Comment:
```suggestion
	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest);
```

This is a safe method.

https://github.com/infinum/eightshift-coding-standards/blob/main/Eightshift/Sniffs/Security/ComponentsEscapeSniff.php#L21-L30